Libs/CodeGen/ApiGen/src/main/resources/api/sys.api

```diff
@@ -59,13 +59,13 @@ api(Sys) {
     @entitle=/repo/read
     @public Map(String, ConnectionInfo) getConnectionInfo(String connectionType);
 
-    [Puts connection info of an instance.]
+    [Delete connection info ]
     @entitle=/repo/write
-    @public void putConnectionInfo(String connectionType, ConnectionInfo connectionInfo);
+    @public void deleteConnectionInfo(String connectionType, ConnectionInfo connectionInfo);
```

Comment:
this should take "String instanceName" as an argument instead of the whole ConnectionInfo object